client/components/web-preview/index.jsx

```diff
@@ -91,6 +93,9 @@ const WebPreview = React.createClass( {
 		}
 		if ( this.props.showPreview ) {
 			document.documentElement.classList.add( 'no-scroll' );
+			this.props.setPreviewShowing( true );
```

Comment:
We don't need the conditional if we just do `this.props.setPreviewShowing( this.props.showPreview )`